compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/WindowState.desktop.kt

```diff
@@ -227,20 +229,25 @@ private class WindowStateImpl(
                     it.position.isSpecified,
                     it.position.x.value,
                     it.position.y.value,
-                    it.size.width.value,
-                    it.size.height.value,
+                    it.size.isSpecified,
```

Comment:
Note for the future.

One of the purposes of this Saver was to save the state on the disc.

As this feature haven't investigated properly and we haven't wrote about it in the docs, it is not guaranteed, and we can break backward compatibility.

If we would officially support it, we would have to move the new field to the end, and check it via `getOrNull`

Reply:
I made it backwards compatible. It's not hard, and better to not ruin someone's day.